pom.xml

```diff
@@ -115,7 +124,7 @@
             </dependency>
             <!-- Pax Logging -->
             <dependency>
-                <groupId>org.ops4j.pax.logging</groupId>
+                <groupId>org.wso2.org.ops4j.pax.logging</groupId>
```

Comment:
This is a major dependency change that might cause regressions.  Do we need to change this under this scope?

Reply:
After bumping the framework version there is an error in building repo unless we update this dependency.